warrior/Framework/ClassUtils/testdata_class.py

```diff
@@ -555,7 +566,19 @@ def resolve_iteration_patterns(self, details_dict):
         cmd_list_length = len(cmd_list)
         cmd_loc_list = [0]
         cmd_size = 1
+        value = ""
         for i, cmd in enumerate(cmd_list):
+            if '+' not in cmd:
+                if value not in cmd and '${' in cmd:
+                    t_obj = TestData()
+                    t_obj.list_substitution_precheck(vc_file_list[i], details_dict, '${', '}')
+                    cmd_value = t_obj.cmd_sub(details_dict, i, vc_file_list[i], '${', '}')
+                    if cmd_value[0] is not False:
+                        t_obj.align_cmd(details_dict, i, cmd_value)
+                        for param in CMD_PARAMS.keys():
+                            del details_dict[param][i]
+            else:
+                value = cmd.split('+')[0]
```

Comment:
Can you check if this logic can be done along with '+' iteration/td_block logic??